schemas/dbt_yml_files.json

```diff
@@ -321,6 +350,26 @@
               "items": {
                 "$ref": "#/$defs/tests"
               }
+            },
+            "versions": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "required": ["v"],
+                "properties": {
+                  "v": { "type": "number" },
+                  "config": { "$ref": "#/$defs/model_configs" },
+                  "columns": {
+                    "type": "array",
+                    "items": {
+                      "anyOf": [
+                        { "$ref": "#/$defs/include_exclude" },
```

Comment:
In the interests of easy future diffs, it might be worth having all these one-line objects spill onto 
```
{
  "multiple": "lines"
},
```

Reply:
bien!